src/qml/PatchManagerPage.qml

```diff
@@ -480,10 +480,11 @@ Page {
                         }
                     }
                     MenuLabel {
-                        visible: patchObject.details.conflicts.length > 0
-                        text: qsTranslate("", "May have conflicts")
+                        visible: !patchObject.details.patched && patchObject.details.conflicts.length > 0
+                        text: qsTr("May conflict with %n other patch(es), see %2", "", patchObject.details.conflicts.length).arg(patchinfoitem.text)
```

Comment:
Interesting use of translation parameter.. Another translation! Let's see how this parameter goes with translations, I have not seen any "Open the Patch info page" or something like that as `%2` as a quick check.

Reply:
Heheh, yeah let's see how this one goes.

And sorry for inconsistently using `qsTr()` instead of `qsTranslate()` here, I wasn't sure the latter actually can do this.